engine/gamesys/src/gamesys/components/comp_collection_proxy.cpp

```diff
@@ -78,14 +78,21 @@ namespace dmGameSystem
         uint32_t                        m_DelayedEnable : 1;
         uint32_t                        m_Unloaded : 1;
         uint32_t                        m_AddedToUpdate : 1;
+        uint32_t                        m_Loading : 1;
 
         dmResource::HPreloader          m_Preloader;
         dmMessage::URL                  m_LoadSender, m_LoadReceiver;
 
         ProxyLoadCallback               m_Callback;
         void*                           m_CallbackCtx;
+        char*                           m_CollectionResPath;  // set from script as an override
     };
 
+    static const char* GetCollectionResorcePath(CollectionProxyComponent* proxy)
+    {
+        return proxy->m_CollectionResPath ? proxy->m_CollectionResPath : proxy->m_Resource->m_DDF->m_Collection;
```

Comment:
Now we check if we have an overridden path to a collection in component itself, if so we return this type instead of one from resource